docker/sirius-lpa-dashboard/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN mkdir -p web/static
 
 COPY package.json .
 COPY yarn.lock .
-RUN yarn
+RUN yarn install --frozen-lockfile --ignore-scripts
```

Comment:
Did you mean yarn ci?

Reply:
Aha, me reading too many related PRs in succession, of course it's yarn not npm.